test/units/module_utils/facts/tests_date_time.py

```diff
@@ -0,0 +1,102 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2020 Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+import pytest
+import datetime
+
+from ansible.module_utils.facts.system.date_time import DateTimeFactCollector
+
+TIMESTAMP = datetime.datetime(2020, 7, 11, 12, 34, 56, 124356)
+
+
+@pytest.fixture
+def fake_now(monkeypatch):
```

Comment:
In Python, it's common to include docstrings as per PEP 257:
```suggestion
def fake_now(monkeypatch):
    """Patch `datetime.datetime.now()` to return a deterministic value."""
```

Reply:
Thank you. Added them.

Reply:
FYI when you accept the suggested changes using GitHub UI, it preserves the authorship of the patch.